docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
+
+You can use OpenRewrite in the {ProductShortName} CLI to prepare Java applications for migration.
+
+[IMPORTANT]
+====
```

Comment:
I think this notice should be moved to the beginning of the section. Otherwise, the stuff that follows, like the sentence before the procedure, gets "lost". If you want to keep it here, then move the "The JavaxtoJakarta recipe" sentence above the TP notice.